### DISCUSSION TITLE
How to use tensorboard while finetuning model ?

### DISCUSSION THREAD

**Discussion by Ca-ressemble-a-du-fake** [coqui-ai/TTS#1154] [General Q&A] [answered]
Hi,
I am following the [fine-tuning tutorial](https://tts.readthedocs.io/en/latest/finetuning.html) to fine-tune French model (Taco2 DDC) but cannot watch how the training evolves with tensorboard.

I found [on this other tutorial](https://lordoapple.medium.com/cloning-your-voice-cb321908b060) how to launch tensorboard (because I am using Colab), but cannot find where to set the log dir for the trainer.

Can somebody explain how to set up the log dir when launching the training ?

**Comment by erogol** [comment, marked as answer]
```
 > EPOCH: 6/1000
 --> /path/to/your/trianing/
...
```

You see something like this on the terminal as you train. So you need to point to that folder to see the training logs.

**Reply by Ca-ressemble-a-du-fake** [reply]
Thank you @erogol but the indicated directory has no event files so I get **No dashboards are active for the current data set.**
The indicated dir only contains 4 files (best.modelXYZ, best.model, config.json, and trainer_0_log.txt. 

So tensorboard event file must be somewhere else or maybe I have to wait although I see in the log "generating test sentences". So they must be somewhere!

**Reply by Ca-ressemble-a-du-fake** [reply]
After some time event file has been written, and tensorboard was updated! So problem solved!

**Reply by erogol** [reply]
good to hear that :)
